### DISCUSSION TITLE
How do I remove these console warnings?

### DISCUSSION THREAD

**Discussion by onurusluca** [nuxt-modules/i18n#2281] [Q&A]
![image](https://github.com/nuxt-modules/i18n/assets/51825926/14b2fc23-12c4-44e0-a3cd-373420d10fe9)

```
  modules: [
    "@nuxtjs/i18n", // https://i18n.nuxtjs.org/
  ],
```

```
 i18n: {
    detectBrowserLanguage: {
      useCookie: false,
      fallbackLocale: "en",
    },
    strategy: "no_prefix",
    locales: [
      { code: "en", iso: "en-US", file: "en.yml", name: "English", dir: "ltr" },
      { code: "ja", iso: "ja-JP", file: "ja.yml", name: "日本語", dir: "ltr" },
    ],
    lazy: true,
    langDir: "lang/",
    defaultLocale: "en",
  },
```

**Comment by BobbieGoede** [comment]
What version do you have installed? The second warning should only be visible if you're using the deprecated hook if you're using `v8.0.0-beta.13` or newer ([since this PR](https://github.com/nuxt-modules/i18n/pull/2093)). 

I think the only way to hide the first warning is by adding [a vueI18n config file](https://v8.i18n.nuxtjs.org/options/vue-i18n) as the module is looking for one by default, I don't think there is another way to disable the warning right now.

**Reply by kazupon** [reply]
we should change to `info`, if we will not config `vueI18n` option.
